pkg/volume/empty_dir/empty_dir_linux.go

```diff
@@ -27,10 +27,12 @@ import (
 const linuxTmpfsMagic = 0x01021994
 
 // realMountDetector implements mountDetector in terms of syscalls.
-type realMountDetector struct{}
+type realMountDetector struct {
+	mounter mount.Interface
```

Comment:
Scratch my comment. I see the usage pattern in other plugins. Its worth being consistent.